### PR TITLE
Add Inkscape

### PR DESCRIPTION
**Pull request by EnderNightLord-ChromeBook**
nightpavell wanted this so here it is

### CODE REVIEW

File: apps/inkscape/inkscape.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+the vector graphics editor
```
Comment (MatthewTighe) [inline apps/inkscape/inkscape.txt]: Use Inkscape's description of their product: "Inkscape is an open-source vector graphics editor similar to Adobe Illustrator, Corel Draw, Freehand, or Xara X. What sets Inkscape apart is its use of Scalable Vector Graphics (SVG), an open XML-based W3C standard, as the native format."

Comment (EnderNightLord-ChromeBook) [reply apps/inkscape/inkscape.txt]: okie dokie...

Comment (EnderNightLord-ChromeBook) [reply apps/inkscape/inkscape.txt]: ok done...

**Comment by EnderNightLord-ChromeBook** [comment]
Ok thats done is it ok to add?

**Comment by EnderNightLord-ChromeBook** [comment]
@MatthewTighe Is it ready to be released into the world?

**Comment by corbinlc** [comment]
There are a few minor things I am going to clean up.  No one will see this until I add it to apps.txt.

**Comment by EnderNightLord-ChromeBook** [comment]
Thanks

**Pull request merged**